ui/app/AppLayouts/stores/RootStore.qml

```diff
@@ -10,11 +13,46 @@ QtObject {
     property var mainModuleInst: mainModule
     property var aboutModuleInst: aboutModule
     property var communitiesModuleInst: communitiesModule
-
     property bool newVersionAvailable: false
     property string latestVersion
     property string downloadURL
 
+    readonly property int loginType: getLoginType()
+    function getLoginType() {
+        if(!userProfileInst)
+            return Constants.LoginType.Password
+
+        if(userProfileInst.usingBiometricLogin)
+            return Constants.LoginType.Biometrics
+        if(userProfileInst.isKeycardUser)
+            return Constants.LoginType.Keycard
+        return Constants.LoginType.Password
+    }
+
+    //TODO see how these values can be retrieved from the community the user attempts to join
+    property var permissionsModel: ChatStore.RootStore.permissionsStore.permissionsModel
```

Comment:
I don't think this will behave correctly. The permissionModel is per chat section, so if you have multiple communities, which model will it use?

I guess that's gonna be addressed in the next PR?

Reply:
@jrainville exactly, together with the others assets, collectibles etc. there is a TODO there for this